vulture/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .core import Vulture   # noqa
```

Comment:
I'd rather not use `noqa` tags in the code. Please use `assert Vulture` instead and leave an empty line after it to separate it from `__version__`.